zerver/webhooks/zapier/doc.html

```diff
@@ -40,3 +40,36 @@
 <p>You're done! Example message:</p>
 
 <img class="screenshot" src="/static/images/integrations/zapier/005.png"/>
+
+<p>
+    <b>Weather Update with Zapier!</b><br />
+</p>
+<p>
+    Go to the following link <code>https://zapier.com/app/explore</code> after you sign into your Zapier account<br />
```

Comment:
I'd change this to:

`Sign into your Zapier account, then go to the URL <code>https://zapier.com/app/explore</code> .`